cuenca/resources/base.py

```diff
@@ -77,6 +79,32 @@ def _update(
         return cls._from_dict(resp)
 
 
+@dataclass
+class Downloadable(Resource):
+    @classmethod
+    def download(
+        cls,
+        instance,
+        *,
+        session: Session = global_session,
+        file_format: FileFormat,
```

Comment:
Is making this a keyword arg by putting it after the `*` intentional? No need to justify. I only want to make sure it wasn't a mistake.

Reply:
Right, file_format should be before *